tests/users/test_resend_email.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import uuid
+from django.urls import reverse
+from django.utils import timezone
+from rest_framework.test import APITestCase
+from unittest.mock import patch
+from users.models import User, UserRole
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
Configuring logging at the test file level may interfere with Django or pytest logging. If logging is necessary, configure it in a test runner or a dedicated conftest.py instead.